var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -26,15 +26,22 @@
 import os
 
 
-class Cantera(Package):
+class Cantera(SConsPackage):
     """Cantera is a suite of object-oriented software tools for problems
     involving chemical kinetics, thermodynamics, and/or transport processes."""
 
     homepage = "http://www.cantera.org/docs/sphinx/html/index.html"
-    url      = "https://github.com/Cantera/cantera/archive/v2.2.1.tar.gz"
+    url      = "https://github.com/Cantera/cantera/archive/v2.3.0.tar.gz"
 
+    version('2.3.0', 'aebbd8d891cb1623604245398502b72e')
```

Comment:
This new release includes a lot of changes to the command-line variables. So far I haven't been able to get it to build properly. If I can't figure it out soon, I might drop it and let someone else deal with it if they need it.